bitcoin.md

```diff
@@ -72,14 +82,14 @@ EOF'
 
 ## Add bitcoin user and set file ownership
 
-```
+```bash
 $ sudo useradd bitcoin
 $ sudo chown -R bitcoin:bitcoin /bitcoin/
 ```
 
 ## Install bitcoin systemd unit
 
-```
+```bash
 $ sudo bash -c 'cat <<EOF> /etc/systemd/system/bitcoin.service
 [Unit]
```

Comment:
not able to properly suggest a change here, but i'd like to add in some conditions to the unit:
```
$ sudo bash -c 'cat <<EOF> /etc/systemd/system/bitcoin.service
[Unit]
Description=Bitcoin daemon
After=network.target
ConditionFileIsExecutable=/usr/local/bin/bitcoind
ConditionPathExists=/bitcoin
ConditionPathExists=/etc/bitcoin/bitcoin.conf
ConditionFileNotEmpty=/etc/bitcoin/bitcoin.conf
```

double check my suggestion, but i think that would work

Reply:
`ConditionPathExists` and `ConditionFileNotEmpty` point to the same file. Is the prior required, or can we only keep the latter instead of checking the same file twice? Other than that looks like a good addition.

Reply:
I think using `ConditionFileNotEmpty` by itself should be good enough, but check the systemd docs to verify

Reply:
Just tested and committed, it works only with that

Reply:
P.S. also added it for stacks (the config part, the `FileIsExecutable` and datadir `PathExists` were already there)

Reply:
Citing systemd [docs](https://www.freedesktop.org/software/systemd/man/latest/systemd.unit.html#:~:text=ConditionFileNotEmpty%3D,non%2Dzero%20size.):
```
ConditionFileNotEmpty=
ConditionFileNotEmpty= is similar to ConditionPathExists= but verifies that a certain path exists and refers to a regular file with a non-zero size.
```